boards/actinius/icarus/feather_connector.dtsi

```diff
@@ -20,8 +20,7 @@
 			   <7 0 &gpio0 21 0>,	/* MOSI */
 			   <8 0 &gpio0 22 0>,	/* MISO */
 			   <9 0 &gpio0 23 0>,	/* RX */
-			   <10 0 &gpio0 24 0>,	/* TX */
-			   /* 11 is VIN */
+			   <10 0 &gpio0 24 0>,	/* TX */	/* 11 is VIN */
```

Comment:
I am currently looking at supporting this sot that no changes are suggested in this case